### PR TITLE
Frank eitt fix

### PR DESCRIPTION
**Pull request by FrankZRS**

### CODE REVIEW

File: api/oc_knx_sec.c [hunk 1]
```diff
@@ -670,6 +670,11 @@ oc_core_auth_at_post_handler(oc_request_t *request,
               char *if_str = oc_string_array_get_item(str_array, i);
               oc_interface_mask_t if_mask =
                 oc_ri_get_interface_mask(if_str, strlen(if_str));
+              if (if_mask == OC_IF_LI) {
+                PRINT("   if.ll is not a valid access scope!\n");
```
Comment (WAvdBeek) [inline api/oc_knx_sec.c:674]: can we do this as a debug statement (OC_DEBUG)?

File: api/oc_knx_client.c [hunk 2]
```diff
@@ -775,7 +774,7 @@ oc_do_s_mode_with_scope_and_check(int scope, const char *resource_url, char *rp,
       PRINT("    handling: index %d\n", index);
       for (int j = 0; j < ga_len; j++) {
         group_address = oc_core_find_group_object_table_group_entry(index, j);
-        PRINT("      ga : %d\n", group_address);
+        PRINT("      ga : %lu\n", group_address);
```
Comment (WAvdBeek) [inline api/oc_knx_client.c:777]: OC_DEBUG?

File: api/oc_knx_sec.c (hunk 1 above)
Comment (WAvdBeek) [reply api/oc_knx_sec.c:674]: OC_DBG

**Review by WAvdBeek — CHANGES_REQUESTED** [review]

File: api/oc_knx_client.c (hunk 2 above)
Comment (FrankZRS) [reply api/oc_knx_client.c:777]: This was a PRINT before, I just fixed the format identifier. There's quite a few more PRINT's in this function, do you want me to change them all?

File: api/oc_knx_sec.c (hunk 1 above)
Comment (FrankZRS) [reply api/oc_knx_sec.c]: Fixed. I used OC_ERR since it's an error that's leading to a BAD_REQUEST response

**Comment by FrankZRS** [comment]
~format

**Comment by WAvdBeek** [comment]
~format

**Pull request merged**